src/trusted/service_runtime/nacl_app_thread.c

```diff
@@ -32,7 +253,7 @@ void WINAPI NaClAppThreadLauncher(void *state) {
 
   NaClSignalStackRegister(natp->signal_stack);
 
-  NaClLog(4, "      natp = 0x%016"NACL_PRIxPTR"\n", (uintptr_t) natp);
+  NaClLog(4, "     natp  = 0x%016"NACL_PRIxPTR"\n", (uintptr_t)natp);
   NaClLog(4, " prog_ctr  = 0x%016"NACL_PRIxNACL_REG"\n", natp->user.prog_ctr);
   NaClLog(4, "stack_ptr  = 0x%016"NACL_PRIxPTR"\n",
```

Comment:
Is there any reason for us to use "4" here in the NaClLog? Other messages are directed to the standard output "1".

Reply:
This particular log message is fairly noisy and not usually useful information, so I have made it require -vvvv to be visible.